frontend/packages/core/src/timeago.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import type { ReactTimeagoProps as TimeAgoProps } from "react-timeago";
-import TimeAgo from "react-timeago";
+import ReactTimeago from "react-timeago";
 
 interface EventTimeProps extends Pick<TimeAgoProps, "date"> {
   onClick?: () => void;
```

Comment:
Should we pick `onClick` from `TimeAgoProps` as well?

Reply:
Picked it, but it made it a required field so I reverted it back for now